src/utils/api/lsk/transactions.js

```diff
@@ -139,7 +156,7 @@ export const broadcast = (transaction, network) => new Promise(
  */
 export const getTransactionBaseFees = () => (
   new Promise(async (resolve) => {
-    const fee = 1e7;
+    const fee = 1e3;
```

Comment:
Where did you get this value?

Reply:
This is just a realistic mock and we will eventually get this from the service.

Reply:
@UsamaHameed  Maybe add a comment that this function will be refactored and refer to the Github ticket number.